Shop Application/src/Shop/Appliance.java

```diff
@@ -0,0 +1,17 @@
+package Shop;
+
+import java.text.ParseException;
+
+public class Appliance extends ElectronicProduct {
+    public Appliance(String name, double price, int quantity, String ageRestriction, int guaranteePeriod) {
```

Comment:
guaranteePeriod nu mai e nevoie sa faca parte din constructor, pentru ca valoarea lui e fixa (6). Oricum parametrul nu este folosit in constructor deci nu isi are rostul.